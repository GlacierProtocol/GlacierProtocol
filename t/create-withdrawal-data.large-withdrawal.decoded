{
  "txid": "b0f186e21777ee30e0031bd6caaa64a06b1a7d29b75345e5e9ca6b384d70d66c",
  "hash": "b0f186e21777ee30e0031bd6caaa64a06b1a7d29b75345e5e9ca6b384d70d66c",
  "version": 2,
  "size": 3247,
  "vsize": 3247,
  "weight": 12988,
  "locktime": 0,
  "vin": [
    {
      "txid": "9a9ae27622141378e12a209d33e6547bbe4188fc92031ba2b2b5c383ce73fb60",
      "vout": 1,
      "scriptSig": {
        "asm": "0 3044022035c19aafa44841cd94f1aedd10046a0e9acf5c3788b56d140a4d9e04f9764cb302203e56724c8b07576b3c211ff499b9d4af57aa839ef42b78a0a9f0fda2eb1d0c29[ALL] 304402206947bb972a5d6a468673ec8c6bae41e38fa545b474c3c3288da999bc44f50e270220561750c9c978517df0c1a41f2552fbc80ed032152ff627d2116aa204960a5222[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "00473044022035c19aafa44841cd94f1aedd10046a0e9acf5c3788b56d140a4d9e04f9764cb302203e56724c8b07576b3c211ff499b9d4af57aa839ef42b78a0a9f0fda2eb1d0c290147304402206947bb972a5d6a468673ec8c6bae41e38fa545b474c3c3288da999bc44f50e270220561750c9c978517df0c1a41f2552fbc80ed032152ff627d2116aa204960a5222014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "1c5d43ef0477f72a3eb2e089bd6cc61ed5fb22b46c7a79910607eab4609da544",
      "vout": 1,
      "scriptSig": {
        "asm": "0 30440220348e4e2cbbb4ac1192a334c65f5a4640612b03291cb387fb248b0254586d19ec02200371494e9e654a9aa8d5e56db154fdd36b2f51113c4629ce8994df0c7ebb14e5[ALL] 3044022069f678a29c01b4393b45f3d72d318ff3dc23774b05f847e7ee410dee0981455b0220082e64454adcd5fd510969322fe85ddb9675d2e9927e47e86edf53895c6fe9a2[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "004730440220348e4e2cbbb4ac1192a334c65f5a4640612b03291cb387fb248b0254586d19ec02200371494e9e654a9aa8d5e56db154fdd36b2f51113c4629ce8994df0c7ebb14e501473044022069f678a29c01b4393b45f3d72d318ff3dc23774b05f847e7ee410dee0981455b0220082e64454adcd5fd510969322fe85ddb9675d2e9927e47e86edf53895c6fe9a2014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "977da2028eac46f33893b695e1bc3fde3a6f95f4704398d7bfd66ae11582c715",
      "vout": 1,
      "scriptSig": {
        "asm": "0 3044022078417ba6fa12826f8f502dcf93242de8f8d9f901166ce6d499374259c5e3ff8f0220304450cec98311af7b0f8e675ef6a9522661fbd05996c85091bd25b82290330b[ALL] 304402204d153354d0dce0059a9966450f9aec5f34b02e1ff634b3423601793f3e9a4a91022005959a5b5825d35875ef102c64133617ad05bc439f35d3a65d5fe4db7bd89669[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "00473044022078417ba6fa12826f8f502dcf93242de8f8d9f901166ce6d499374259c5e3ff8f0220304450cec98311af7b0f8e675ef6a9522661fbd05996c85091bd25b82290330b0147304402204d153354d0dce0059a9966450f9aec5f34b02e1ff634b3423601793f3e9a4a91022005959a5b5825d35875ef102c64133617ad05bc439f35d3a65d5fe4db7bd89669014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "6932dec4f79eef0298d97cbbdda1ef0d536bd3d6c74298164317aa3ab6a4a705",
      "vout": 0,
      "scriptSig": {
        "asm": "0 30440220200165125efde11523eb5f153a2deb9cefaa9d64cb14d47ccac860abf38f48f3022012ea646d2c244bde925300480e8ca16af342e7264ee582ff3a207cd835c8a6cd[ALL] 3044022073687f63256210ff11b8e3485adc9be246963ca2d5ce89e4e5558493407a812902203d465e573c53c6ab962ea66811398aa3b40847873e8526eecf8b2865c2bdf11f[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "004730440220200165125efde11523eb5f153a2deb9cefaa9d64cb14d47ccac860abf38f48f3022012ea646d2c244bde925300480e8ca16af342e7264ee582ff3a207cd835c8a6cd01473044022073687f63256210ff11b8e3485adc9be246963ca2d5ce89e4e5558493407a812902203d465e573c53c6ab962ea66811398aa3b40847873e8526eecf8b2865c2bdf11f014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "efcb661502b71ef0c0e5bad0faeeb853e322bc17903e173e424db4b8815dc5a1",
      "vout": 1,
      "scriptSig": {
        "asm": "0 304402202a8cf96f42d1607b265c2afb56f2d568129cc904a3b12ed6f08740de5b8fc67f02204b5705bce9cce873faf25c2f63b7930c01adb300615bb4a28920fa6d9f3d5917[ALL] 30440220183fa0a62252d5dc375739a144b2ec56152a4f83760fd7f4543e4529ca8abac802206cf6de290e5c3430535220a94ec2f0fc3b74f8266034a5742751c7c0a70e3c7a[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "0047304402202a8cf96f42d1607b265c2afb56f2d568129cc904a3b12ed6f08740de5b8fc67f02204b5705bce9cce873faf25c2f63b7930c01adb300615bb4a28920fa6d9f3d5917014730440220183fa0a62252d5dc375739a144b2ec56152a4f83760fd7f4543e4529ca8abac802206cf6de290e5c3430535220a94ec2f0fc3b74f8266034a5742751c7c0a70e3c7a014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "c6ef0ea8f8bf3cbb0c173cb3a3446d664bd3ff026864492bd70004f8210cd4c1",
      "vout": 0,
      "scriptSig": {
        "asm": "0 3044022074aee24ec00a574138405408d0ad35d0ffe94e3cdc23a1fa2210309a5c228386022073e57cd01ed2c400d016a95f02c902511b6682b75563ab958035e9c20ebb9f2c[ALL] 304402203565e9fc32c2ab284377c66778dc638d64caca9bb5d116b693884a3106e75d9502202357c5cdaf8c68a427846256c716e50cfa65de26cae3168e32f8a1ffafa3c883[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "00473044022074aee24ec00a574138405408d0ad35d0ffe94e3cdc23a1fa2210309a5c228386022073e57cd01ed2c400d016a95f02c902511b6682b75563ab958035e9c20ebb9f2c0147304402203565e9fc32c2ab284377c66778dc638d64caca9bb5d116b693884a3106e75d9502202357c5cdaf8c68a427846256c716e50cfa65de26cae3168e32f8a1ffafa3c883014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    },
    {
      "txid": "ae564d46e3309c340a67b32c0bf72dfa68e4f0cf758891c52110a2d88ea4e7e4",
      "vout": 0,
      "scriptSig": {
        "asm": "0 3044022006a567bcd31761535a6231549dce4b412e943272656f2672e800be3bf885253c0220464d46b24ababb3c05ce783ec98e532dd15c05ea67515f93acbe6a184d267e72[ALL] 30440220434746838780c09fd12ba9f83691ccd196599563cc8d166fc9587eb35f18200d022074e95e4b04737d228dd62aafbdb14ce9d6a516da55e79cf13858809a81d24a8e[ALL] 52410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae",
        "hex": "00473044022006a567bcd31761535a6231549dce4b412e943272656f2672e800be3bf885253c0220464d46b24ababb3c05ce783ec98e532dd15c05ea67515f93acbe6a184d267e72014730440220434746838780c09fd12ba9f83691ccd196599563cc8d166fc9587eb35f18200d022074e95e4b04737d228dd62aafbdb14ce9d6a516da55e79cf13858809a81d24a8e014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54ae"
      },
      "sequence": 4294967295
    }
  ],
  "vout": [
    {
      "value": 0.07226721,
      "n": 0,
      "scriptPubKey": {
        "asm": "0 1b8dffa18bc3e70d2aadf872d396b6b241a79e5b",
        "hex": "00141b8dffa18bc3e70d2aadf872d396b6b241a79e5b",
        "reqSigs": 1,
        "type": "witness_v0_keyhash",
        "addresses": [
          "tb1qrwxllgvtc0ns624dlped894kkfq608jmxsnmrx"
        ]
      }
    }
  ]
}
