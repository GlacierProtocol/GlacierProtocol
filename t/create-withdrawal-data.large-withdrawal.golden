Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
You will need to enter several pieces of information to create a withdrawal transaction.


*** PLEASE BE SURE TO ENTER THE CORRECT DESTINATION ADDRESS ***


Source cold storage address: 
Redemption script for source cold storage address: 
Destination address: 
How many unspent transactions will you be using for this withdrawal? 
Please paste raw transaction #1 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Please paste raw transaction #2 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Please paste raw transaction #3 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Please paste raw transaction #4 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Please paste raw transaction #5 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Please paste raw transaction #6 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Please paste raw transaction #7 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Transaction data found for source address.
TOTAL unspent amount for this raw transaction: 0.07230000 BTC

How many private keys will you be signing this transaction with? 
#: Key #1: Key #2: 
Enter fee rate.
Satoshis per vbyte: 
Based on the provided rate, the fee will be 0.00003279 bitcoin.
Confirm? (y/n): 
Please enter the decimal amount (in bitcoin) to withdraw to the destination address.

Example: For 2.3 bitcoins, enter "2.3".

After a fee of 0.00003279, you have 0.07226721 bitcoins available to withdraw.

*** Technical note for experienced Bitcoin users:  If the withdrawal amount & fee are cumulatively less than the total amount of the unspent transactions, the remainder will be sent back to the same cold storage address as change. ***

Amount to send to tb1qrwxllgvtc0ns624dlped894kkfq608jmxsnmrx (leave blank to withdraw all funds stored in these unspent transactions): 
Is this data correct?
*** WARNING: Incorrect data may lead to loss of funds ***

0.07230000 BTC in unspent supplied transactions
0 BTC going back to cold storage address 2MyCc3oEVwSYzgmspKTjZ6HZrzeg8ajdmKR
0.07226721 BTC going to destination address tb1qrwxllgvtc0ns624dlped894kkfq608jmxsnmrx
Fee amount: 0.00003279

Signing with private keys: 
93VDtjySSuPjgLYyCR6BBTqCuaqu6X728ZfkNEDbgAdVURRd1rS
924WjH7yMYyuyhUougBHEu2vThcHRmahF6FsKvNVQBwSGuEyqKm


Confirm? (y/n): 
Calculating transaction...


Sufficient private keys to execute transaction?
True

Raw signed transaction (hex):
020000000760fb73ce83c3b5b2a21b0392fc8841be7b54e6339d202ae17813142276e29a9a01000000fd9f0100473044022035c19aafa44841cd94f1aedd10046a0e9acf5c3788b56d140a4d9e04f9764cb302203e56724c8b07576b3c211ff499b9d4af57aa839ef42b78a0a9f0fda2eb1d0c290147304402206947bb972a5d6a468673ec8c6bae41e38fa545b474c3c3288da999bc44f50e270220561750c9c978517df0c1a41f2552fbc80ed032152ff627d2116aa204960a5222014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff44a59d60b4ea070691797a6cb422fbd51ec66cbd89e0b23e2af77704ef435d1c01000000fd9f01004730440220348e4e2cbbb4ac1192a334c65f5a4640612b03291cb387fb248b0254586d19ec02200371494e9e654a9aa8d5e56db154fdd36b2f51113c4629ce8994df0c7ebb14e501473044022069f678a29c01b4393b45f3d72d318ff3dc23774b05f847e7ee410dee0981455b0220082e64454adcd5fd510969322fe85ddb9675d2e9927e47e86edf53895c6fe9a2014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff15c78215e16ad6bfd7984370f4956f3ade3fbce195b69338f346ac8e02a27d9701000000fd9f0100473044022078417ba6fa12826f8f502dcf93242de8f8d9f901166ce6d499374259c5e3ff8f0220304450cec98311af7b0f8e675ef6a9522661fbd05996c85091bd25b82290330b0147304402204d153354d0dce0059a9966450f9aec5f34b02e1ff634b3423601793f3e9a4a91022005959a5b5825d35875ef102c64133617ad05bc439f35d3a65d5fe4db7bd89669014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff05a7a4b63aaa1743169842c7d6d36b530defa1ddbb7cd99802ef9ef7c4de326900000000fd9f01004730440220200165125efde11523eb5f153a2deb9cefaa9d64cb14d47ccac860abf38f48f3022012ea646d2c244bde925300480e8ca16af342e7264ee582ff3a207cd835c8a6cd01473044022073687f63256210ff11b8e3485adc9be246963ca2d5ce89e4e5558493407a812902203d465e573c53c6ab962ea66811398aa3b40847873e8526eecf8b2865c2bdf11f014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffa1c55d81b8b44d423e173e9017bc22e353b8eefad0bae5c0f01eb7021566cbef01000000fd9f010047304402202a8cf96f42d1607b265c2afb56f2d568129cc904a3b12ed6f08740de5b8fc67f02204b5705bce9cce873faf25c2f63b7930c01adb300615bb4a28920fa6d9f3d5917014730440220183fa0a62252d5dc375739a144b2ec56152a4f83760fd7f4543e4529ca8abac802206cf6de290e5c3430535220a94ec2f0fc3b74f8266034a5742751c7c0a70e3c7a014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffc1d40c21f80400d72b49646802ffd34b666d44a3b33c170cbb3cbff8a80eefc600000000fd9f0100473044022074aee24ec00a574138405408d0ad35d0ffe94e3cdc23a1fa2210309a5c228386022073e57cd01ed2c400d016a95f02c902511b6682b75563ab958035e9c20ebb9f2c0147304402203565e9fc32c2ab284377c66778dc638d64caca9bb5d116b693884a3106e75d9502202357c5cdaf8c68a427846256c716e50cfa65de26cae3168e32f8a1ffafa3c883014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffffe4e7a48ed8a21021c5918875cff0e468fa2df70b2cb3670a349c30e3464d56ae00000000fd9f0100473044022006a567bcd31761535a6231549dce4b412e943272656f2672e800be3bf885253c0220464d46b24ababb3c05ce783ec98e532dd15c05ea67515f93acbe6a184d267e72014730440220434746838780c09fd12ba9f83691ccd196599563cc8d166fc9587eb35f18200d022074e95e4b04737d228dd62aafbdb14ce9d6a516da55e79cf13858809a81d24a8e014d0b0152410498137b6223a9fa395e16fe6e49a7f4a737029d4476ec35844a8469e358d641b85a5bf355d570699d63cde87da1d450e1966d97ee61a99b51995a11eb7ea0f6744104fd9a69f95e31c3b8d2c97925e0f151dd1c67f3b71e3ba6db8f20d944e6e94d9b5e3ad4c12a5f9bd6d2137cf69b015f9637de61aafae29fdfccf8ef5a4cd81e374104ff188a131ee79900a4cdbe68c5ea84ad73b8acf8414e5895f6c7b0e4779ef50d39ba369a34cd3f7216877769a292ccf52c2653c799a138321bf8c1e03a9120594104eeafc0621008662a9977b953ee7f7af87981830f561cac50387211c75d58bcb751414fc9abe0cf4a28a4044806ccd54b3f80ba990e0eff046bb1b930968b973a54aeffffffff0161456e00000000001600141b8dffa18bc3e70d2aadf872d396b6b241a79e5b00000000

Transaction fingerprint (md5):
4989f7e89d72a99d90bd641da304d5ea
QR code for transaction written to transaction-01.png,transaction-02.png
transaction-01.png
transaction-02.png
